icons/book-spine.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
Not sure about the use case for this one either.

Reply:
I my comment on `book-elastic` applies here on this one as well. I think with `book` icon plus one line in it. It is hard to say if it will work, probably `book` will work as well.